package.json

```diff
@@ -31,6 +32,12 @@
     "test-watch": "npm test -- --watch"
   },
   "browserslist": "electron 1.6",
+  "lint-staged": {
+    "*.js": [
+      "prettier --single-quote --write",
```

Comment:
Shouldn't `lint-staged` auto fix with eslint as well?

Reply:
Yes, it can add `eslint --fix`. 
But I think what `eslint --fix` can do as same as prettier(something like `add semicolon` or `double quotes to single quote`) .
I don't known what error eslint can auto-fix but prettier can't.

This is the [eslint official doc](https://eslint.org/docs/user-guide/command-line-interface#--fix) said, but I don't understand what it mean.

If `eslint --fix` has the same effect as prettier, I think keep only prettier is better.